tools/bin/go_core_tests

```diff
@@ -2,15 +2,10 @@
 set -o pipefail
 set +e
 
-go test -v -p 4 -parallel 4 ./... >./output.txt
-EXITCODE=$?
-
 echo "Failed tests and panics: ---------------------"
 echo ""
-grep --line-number --after-context=2 "panic: " output.txt
-echo ""
-grep --line-number "\-\-\- FAIL" output.txt
-grep -n "FAIL	" output.txt
+go test -v -p 4 -parallel 4 ./... | tee ./output.txt | grep --line-buffered --line-number --after-context=2 -e "panic: " -e "\-\-\- FAIL" -e "FAIL " -e "driver: bad connection"
```

Comment:
It's a bit noisier because of the after-context, which was previously only used on `panic:`

Reply:
@archseer `grep "driver: bad connection" output.txt` was on purpose separated to be under  the horizontal line and "Potentially related:" - because usually it's not the direct cause of failure. So would be good to keep it there, or remove completely

Reply:
I think it's useful to keep because I've been getting that as a cause of failure quite frequently. I can separate it back below the line?

Reply:
Yeah, would be good to separate it. Hm let me know if you see it as a cause, i'd like to have a look